WebRoot/public/login.jsp

```diff
@@ -16,7 +17,7 @@
 <fmt:setBundle basename="/messages/ZMsg" var="zmsg" scope="request"/>
 
 <%-- query params to ignore when constructing form port url or redirect url --%>
```

Comment:
around line 13,
the change of commit 4cde664d75ba7345f7c77205a1dc5b6da104f55a on snickers (6e69efad8fa0911aa478d2cbb4304c543b8f097f on bugfic/NICPS-155) seems to be removed at solving conflicts.
Is it expected change?

Reply:
Hi Koichi
I verified this on 8.8.8 and NICPS-155-check branch, but I have same code you had in your 'check' branch.
Please confirm if this commit '4cde664' need to add .

Reply:
The branch NICPS-155-check doesn't contain 4cde664 but bugfic/NICPS-155 contained it. Please compare branches carefully checking commit log and graph.